README.md

```diff
@@ -100,6 +101,26 @@ General security best practices apply:
   * Ultimately, the security of a system depends on the capabilities of its administrator.
   * Care should be taken when installing new software; only install from official sources that the developers indicate on their official website/github/etc.
 
+# Threat Modeling
+
+The first and most important step for security and privacy is to create a threat model. After all, how can we have security when we don't even know what we're securing against? Each person will have their own needs so everyone's threat model will be different. Threat models tend to evolve over time as our situation changes, so be sure to periodically reassess your threat model.
+
+Step 1: Identify what you're protecting
+
+This is probably a lot of things: your phone, your laptop, passwords stored on your devices, internet browsing history, etc. Make a list of all the things you can think of that you want to protect and store it somewhere.
+
+Step 2: Identify threat/s
```

Comment:
I would call this "Identify adversaries" not threats.

Reply:
Changed